chromium_src/media/audio/pulse/pulse_util.cc

```diff
@@ -3,7 +3,48 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
-constexpr char kBrowserDisplayName[] = "brave-browser";
+#include "base/strings/string_util.h"
+#include "build/build_config.h"
+
+#define STABLE_DISPLAY_NAME "brave-browser"
+#define DEV_DISPLAY_NAME "brave-browser-dev"
+#define BETA_DISPLAY_NAME "brave-browser-beta"
+#define NIGHTLY_DISPLAY_NAME "brave-browser-nightly"
+
+namespace media {
+
+namespace pulse {
+
+namespace {
+    // We can't use brave::GetChannelName() because audio is a dependency of
+    // common, so we implement minimal channel checking here to determine which
+    // icon to use. This is Linux only so it's fine just using those modifiers.
+    //
+    // The defines are pulled in via a very small patch to the audio BUILD.gn.
+    const char* BrowserDisplayName() {
```

Comment:
Perhaps just a nit, but I'd expect `BrowserDisplayName` to also affect the displayed name rather than only the icon.

Reply:
Renamed to `BrowserIconName()`